pages/index.tsx

```diff
@@ -11,7 +12,8 @@ import Button from '@mui/material/Button';
 const MainApp = () => {
   const { user, error, isLoading } = useUser();
   const [zone, setZone] = useState<InterestZone>();
-  const [openModal, setOpenModal] = useState(false);
+  const [openEditModal, setOpenEditModal] = useState(false);
+  const [openAddModal, setOpenAddModal] = useState(false);
```

Comment:
Please rename both state using the following convention 
`
const [editModalVisible, setEditModalVisible] = useState(false);
const [addModalVisible, setAddModalVisible] = useState(false);
`